src/extensions/extensionInterface.ts

```diff
@@ -289,7 +295,14 @@ export const recordingSelectionReducer: Reducer<RecordingSelection, RecordingSel
     else if (action.type === 'SetSelectedElectrodeIds') {
         return {
             ...state,
-            selectedElectrodeIds: action.selectedElectrodeIds
+            selectedElectrodeIds: action.selectedElectrodeIds.filter(eid => ((!state.visibleElectrodeIds) || (state.visibleElectrodeIds.includes(eid))))
```

Comment:
Enforce that selected electrode IDs must be among the visible IDs

Reply:
"or else they stop being selected," right? vs. "or else they're added to the visible set"

Reply:
yes, the first.